db/migrate/7_add_account_type_to_accounts.rb

```diff
@@ -0,0 +1,15 @@
+class AddAccountTypeToAccounts < ActiveRecord::Migration
+  def change
+    add_column :openstax_accounts_accounts, :account_type, :integer
+
+    OpenStax::Accounts::Account.find_each do |account|
+      account.set_account_type
```

Comment:
Wouldn't happen automatically for existing accounts until they were updated somewhere else.